clients/destination/v0/destination.go

```diff
@@ -264,7 +264,7 @@ func (c *Client) Write(ctx context.Context, source string, syncTime time.Time, r
 		if err := saveClient.Send(&pb.Write_Request{
 			Resource:  resource,
 			Source:    source,
-			Timestamp: timestamppb.New(syncTime),
+			Timestamp: timestamppb.New(syncTime.Truncate(time.Microsecond)),
```

Comment:
Given that this is looped for every resource, maybe `syncTime = syncTime.Truncate(time.Microsecond)` at the very top of the method? (In fact every "syncTime-method" should be set up like that, in case `syncTime` was used a second time somewhere else and we forget to truncate it again)

Reply:
Updated the pr